docs/release-testing.md

```diff
@@ -1,137 +1,70 @@
 ## Generating and testing release bundles
 
-A full overview of community operator testing can be found [here](https://operator-framework.github.io/community-operators/testing-operators/). The following steps were parsed from the testing document and are specific to testing the `koku-metrics-operator`.
-
 ### Pre-requisites
 
-* Access to a 4.5+ OpenShift cluster
-* opm
-
-To install opm, clone the operator-registry repository:
-
-```
-git clone https://github.com/operator-framework/operator-registry
-```
-Change into the directory and run `make build`. This will generate an `opm` executable in the `operator-registry/bin/` directory. Add the bin to your path, or substitute `opm` in the following commands with the full path to your `opm` executable.
-
+* Access to an OpenShift cluster (any currently supported version)
 
 ### Testing an operator upgrade - Overview
 
-Testing an upgrade is a complicated and an involved process. The general steps are as follows:
-1. Create an opm index (test-catalog) for the most recent release and push to Quay (quay.io/project-koku/kmc-test-catalog contains previous versions and can be used instead of creating your own initial catalog).
-2. Create a CatalogSource in OCP, and install the operator (this should be the version that will be upgraded).
+Reference: https://sdk.operatorframework.io/docs/olm-integration/tutorial-bundle
+Openshift comes with OLM enabled, so the "Enable OLM" steps can be omitted.
+
+Testing an upgrade is composed of the following general steps:
+1. Create a namespace (koku-metrics-operator)
+2. Install the previous bundle (make bundle-deploy-previous)
 3. Generate the new controller image, and push to Quay.
 4. Generate the new bundle.
 5. Build the bundle image and push to Quay.
-6. Create an opm index (test-catalog) that contains the last release and the new release, and push to Quay.
-7. Update the CatalogSource in OCP to point to the new test-catalog image.
-8. Observe the installed operator, and ensure it upgrades automatically.
+6. Deploy the upgraded bundle (make bundle-deploy-upgrade)
+7. Observe the installed operator, and ensure it upgrades automatically.
```

Comment:
If you make all of the ordered list items a 1, the order will be calculated automatically.
```suggestion
1. Create a namespace (koku-metrics-operator)
1. Install the previous bundle (make bundle-deploy-previous)
1. Generate the new controller image, and push to Quay.
1. Generate the new bundle.
1. Build the bundle image and push to Quay.
1. Deploy the upgraded bundle (make bundle-deploy-upgrade)
1. Observe the installed operator, and ensure it upgrades automatically.
```